webapp/components/wpt-metadata.js

```diff
@@ -203,11 +203,9 @@ class WPTMetadata extends PathInfo(LoadingState(PolymerElement)) {
     for (const test of Object.keys(metadata).filter(k => this.shouldShowMetadata(k, path, testResultSet))) {
```

Comment:
It would be useful to document this method, either just before the loop or adding method-level documentation. Something like:

```
    // This loop constructs both the metadataMap, which is used to show inline
    // bug icons in the test results, and displayedMetdata, which is the list of
    // metadata links shown at the bottom of the page.
    let metadataMap = {};
    ....
```